dev/com.ibm.ws.kernel.boot.core/src/com/ibm/ws/kernel/boot/internal/commands/PackageProcessor.java

```diff
@@ -170,6 +172,14 @@ protected File buildManifestForIncludeHasRunnable(File installationManifest) thr
         mf.getMainAttributes().putValue("Main-Class", "wlp.lib.extract.SelfExtractRun");
         mf.getMainAttributes().putValue("Server-Name", processName);
 
+        // For Java 9, we need to apply the /wlp/lib/platform/java/java9.options to the manifest.
+        if (!System.getProperty("java.specification.version").startsWith("1.")) {
```

Comment:
You might want to check for null here.. java.specification.version should always be there, but probably best to be safe

Reply:
I will add that null check.